sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationLiveTests.cs

```diff
@@ -38,14 +38,21 @@ private string GenerateSnapshotName(string prefix = "snapshot-")
             return prefix + Recording.GenerateId();
         }
 
-        private ConfigurationClient GetClient()
+        private ConfigurationClient GetClient(bool skipClientInstrumentation = false)
```

Comment:
Had to skip client instrumentation to make tests work due to a limitation of the Test Framework. Because of this I had to add separate methods for sync and async methods.